impeller/playground/backend/vulkan/playground_impl_vk.cc

```diff
@@ -73,7 +73,7 @@ PlaygroundImplVK::PlaygroundImplVK(PlaygroundSwitches switches)
     return;
   }
 
-  InitGlobalVulkanInstance();
+  // InitGlobalVulkanInstance();
```

Comment:
commented out code

Reply:
Didn't intend to check this in, this was to locally make it possible to run some of the vulkan tests on mac. Reverted.